spec/lita/handlers/totems_spec.rb

```diff
@@ -123,6 +124,26 @@ def send_destroy_message
         end
       end
 
+      context "with a timeout" do
```

Comment:
can we add a test to show it doesnt create a timeout key if totem is not a dev one?

Reply:
Yeah, I'll try thinking of a way to do it, maybe mocking the start function, because currently there is no way to know if the thread has been called

Reply:
maybe we can have a dev_env_array and check the logic when using a totem from that array and when using a different totem (same logic you just added) https://github.com/change/lita-totems/pull/1/files#diff-868c63eb562ba03146634af0ff2cbf9e892e7fd022098d3ae88dd01511befe94R9